pkg/cleantemp/cleantemp.go

```diff
@@ -44,38 +51,35 @@ func CleanTempDir(ctx logContext.Context, dirName string, pid int) error {
 	}
 
 	for _, proc := range procs {
-		if strings.Contains(proc.Executable(), dirName) {
+		if proc.Executable() == execName {
 			pids = append(pids, strconv.Itoa(proc.Pid()))
 		}
 	}
 
 	tempDir := os.TempDir()
-	files, err := os.ReadDir(tempDir)
+	dirs, err := os.ReadDir(tempDir)
 	if err != nil {
 		return fmt.Errorf("Error reading temp dir: %w", err)
 	}
 
-	// Current PID
-	pidStr := strconv.Itoa(pid)
-
 	pattern := `^trufflehog-\d+-\d+$`
```

Comment:
This pattern is hard-coded to use `trufflehog`, should it be using `execName`? Sorry I'm not familiar with the naming convention we decided on.

Reply:
No, MkdirTemp has "trufflehog" hardcoded to create the temp directories.